create-custom-map-style/tileserver/example-style-descriptor.json

```diff
@@ -0,0 +1,7381 @@
+{
```

Comment:
This is redundant with the style descriptor in `public/`, yes?

Reply:
I suppose so. The reason I added it in two places is because we have a "before" and "after" version. One will reference localhost for styling the map with tessera and the other will reference Location Service once it's ready to be bundled in the react app.

Reply:
Ah, gotcha. That makes total sense.